bld/namelist_files/namelist_defaults_dust_emis.xml

```diff
@@ -14,7 +14,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 -->
 
 <!-- Defaults for dust_emis_inparm namelist  -->
-<dust_emis_method>Zender_2003</dust_emis_method>
+<dust_emis_method>Leung_2023</dust_emis_method>
```

Comment:
As above it should be Zender_2003 for anything except clm6_0 physics which is then Leung_2023